File: 001-100/002/evenFibonacciNumbers.js
function evenFibonacciNumbers(){
  // Setup
  let n = document.getElementById("inputN").value;
  let fib = [1, 2];
  // Generate Fibonacci Sequence
  for(let i = 2; i < n; i++){
    if(fib[i - 1] + fib[i - 2] > n){ break; }
    else{ fib.push(fib[i - 1] + fib[i - 2]); }
  }
  // Calculate Even Sum
  let sum = 0;
  fib.forEach(function(element){
    if(element % 2 == 0) { sum += element; }
  });
  // Display Answer
  let answer = `Sum of even-valued fibonacci terms below ${n}<br>${sum}`;
  document.getElementById("answer").innerHTML = answer;
}

File: 001-100/002/evenFibonacciNumbers.py
# Generate the Fibonacci list, stopping at given value
def maxFibonacciValue(n):
    fib = [1, 2]
    for i in range(2, int(n)):
        if(fib[i-1] + fib[i-2] > int(n)):
            break
        else:
            fib.append(fib[i-1] + fib[i-2])
    return fib

# Calculate even-valued sum
def evenFibonacciNumbers(n):
    sum = 0
    for value in maxFibonacciValue(n):
        if(value % 2 == 0):
            sum += value
    return sum

if __name__ == "__main__":
    n = input("Enter a max fibonacci value n: ")
    print("Sum of even-valued fibonacci terms below {:,}".format(int(n)))
    print("{:,}".format(evenFibonacciNumbers(n)))